### DISCUSSION TITLE
vscode debugging together with Godot Inspector on Remote Scene Tree

### DISCUSSION THREAD

**Discussion by pavibear** [godotengine/godot-vscode-plugin#642] [Q&A] [answered]
With v2.0.0, is it possible to use vscode debugging and at the same time use the Godot instead of vscode Inspector to look at the Remote/Active Scene Tree?
If yes, what changes to the vscode and Godot configurations would be required?

**Comment by DaelonSuzuka** [comment, marked as answer]
No.
